### PR TITLE
My expenses page - Stats change on selection

### PR DESCRIPTION
**Pull request by lakshmisuhas1996**
Loom video - https://www.loom.com/share/0092a5a5e40348a5bdd73ba3e1db49fd

### CODE REVIEW

File: src/app/fyle/my-expenses/my-expenses.page.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
     </ion-title>
     <ion-buttons mode="md" slot="end">
       <ng-container *ngIf="myExpenses$ | async as list">
-        <ion-button (click)="switchSelectionMode()" *ngIf="list.length" class="text-capitalize" color="primary">
+        <ion-button (click)="switchSelectionMode()" *ngIf="list.length && isConnected$|async" class="text-capitalize" color="primary">
```
Comment (madhavmansuriya40) [inline src/app/fyle/my-expenses/my-expenses.page.html:19]: please add comments here why this change is required

**Review by madhavmansuriya40 — APPROVED** [review]

File: src/app/fyle/my-expenses/my-expenses.page.html (hunk 1 above)
Comment (lakshmisuhas1996) [reply src/app/fyle/my-expenses/my-expenses.page.html:19]: making this change because in offline mode the select option on top is still visible and on selecting - the bottom stats will toggle and become zero; so showing select option only in online mode and have expenses

**Review by iamtarun1993 — APPROVED** [review]

**Review by abhishek1234321 — APPROVED** [review]
